README.md

```diff
@@ -2,36 +2,43 @@
 
 ## Overview
 
-A brief description. Be clear about the purpose of the resolver,
-its capabilities and limitations.
+A custom Jinja filter for Sceptre for unquoting resolvers appearing in var files.
 
 ## Installation
 
 Installation instructions
 
 To install directly from PyPI
 ```shell
-pip install sceptre-resolver-template
+pip install jinja-unquote-resolvers-filter
 ```
 
 To install from the git repo
 ```shell
-pip install git+https://github.com/Sceptre/sceptre-resolver-template.git
+pip install git+https://github.com/Sceptre/jinja-unquote-resolvers-filter.git
 ```
 
 ## Usage/Examples
 
-Refer to the [Sceptre resolver docs](https://docs.sceptre-project.org/latest/docs/resolvers.html#resolvers)
-for more information.
+In your var file:
 
 ```yaml
-parameter|sceptre_user_data:
-  <name>: !custom <args>
+Subnets:
+  - '!stack_output_external mystack::subnet_a'  # Quotes needed to protect a YAML tag.
+  - '!stack_output_external mystack::subnet_b'
+
+VPC: '!stack_output_external mystack::vpc_id'
 ```
 
-## Example
+In your config:
 
 ```yaml
-parameters:
-  message: !custom "HelloWorld"
+j2_environment:
+  extensions:
+    - jinja_unquote_resolvers_filter.UnquoteResolversFilterExtension
+
+sceptre_user_data:
+  subnets:
+    {{ var.Subnets | unquote_resolvers(output_indent=4) | trim }}
```

Comment:
I wonder why the trimming functionality isn't automatically built into your filter. It seems like including that functionality would make your filter easier to use.

Reply:
I don't agree. We won't always want trimming to occur and the filter should "do one thing well". `to_nice_yaml` doesn't trim for example.

Reply:
On looking at this further, I could not reproduce any scenario where trimming is needed in the first place, so I updated the README to remove it.

Reply:
On actually testing it, I confirmed there is in fact a need for trim, and so now have implemented a trim mode internally.